docs/canary/concepts/islands.md

```diff
@@ -40,12 +40,15 @@ const app = new App()
 Passing props to islands is supported, but only if the props are serializable.
 Fresh can serialize the following types of values:
 
-- Primitive types `string`, `number`, `boolean`, `bigint`, and `null`
-- `Infinity`, `-Infinity`, and `NaN`
+- Primitive types `string`, `number`, `boolean`, `bigint`, `undefined`, and
+  `null`
+- `Infinity`, `-Infinity`, `-0`, and `NaN`
```

Comment:
Added `-0` since the other definitions here are very specific too, so added it for completeness.